tests/parsers/beckman_pharmspec/beckman_pharmspec_parser_test.py

```diff
@@ -133,17 +155,30 @@ def test_get_model(test_file: Path) -> None:
             )
             assert test.value == particle_size
 
+    if has_calculated_doc:
+        assert (
+            model.light_obscuration_aggregate_document.calculated_data_aggregate_document
+        )
+    else:
+        assert (
+            not model.light_obscuration_aggregate_document.calculated_data_aggregate_document
+        )
+
 
 @pytest.mark.short
-def test_asm(test_file: Path) -> None:
+@pytest.mark.parametrize("file_name", OUTPUT_FILES)
+def test_asm(file_name: str) -> None:
```

Comment:
This isn't new in your PR, but we should remove this test, as it's redundant.

The test below calls `allotrope_from_file` and actually checks contents, while this just checks that we could process the file at all.

Reply:
Done